pyccl/tests/test_pkhm.py

```diff
@@ -138,6 +138,10 @@ def f(k, a):
                            'h2': True, 'itg': 'spline',
                            'p2': None},
                           {'cv': None, 'norm': True,
+                           'pk': 'linear', 'h1': True,
```

Comment:
Why are you adding one extra test here? It seems what you added is just a repetition of the same test, right?

Reply:
This addition is irrelevant to my PR. I added one extra test to cover for an extra line of code that wasn't covered before (when there's a `prof2` and `normprof2==False`).

Reply:
Oh, I see. Didn't realise `norm` wasn't the same in both cases. OK, good.